pkgs/games/sm64ex/default.nix

```diff
@@ -53,6 +53,8 @@ stdenv.mkDerivation rec {
     cp build/${region}_pc/sm64.${region}.f3dex2e $out/bin/sm64ex
   '';
 
+  enableParallelBuilding = true;
```

Comment:
Makes the build significantly faster on multi-core systems (specially important for a derivation that is never cached on Hydra).